README.md

```diff
@@ -1,25 +1,33 @@
 # The ArangoDB Cloud
 
-![ArangoDB Oasis](https://cloud.arangodb.com/assets/logos/arangodb-oasis-logo-whitebg-right.png)
+![ArangoGraph](./resources/arangograph.jpeg)
 
-ArangoDB Oasis is the easiest way to run ArangoDB.
+The ArangoGraph Platform is the easiest way to run ArangoDB in the cloud.
 
-Oasis is built, monitored and managed by the cloud specialists of ArangoDB.
-Oasis lets you easily scale from small deployments to very large
-deployments spanning dozens of nodes at any time.
+**Getting started**
 
-## Getting started
+Take ArangoGraph for a free 14-day test ride on [cloud.arangodb.com](https://cloud.arangodb.com). No credit card required.
```

Comment:
We should update this url as well, but can be done in seperate PR

Reply:
ty fixed in [09c2065](https://github.com/arangodb/cloud/pull/4/commits/09c2065a65caeaaa354b309d07e81139b11971c7)